src/Arcus.Templates.Tests.Integration/WebApi/WebApiProject.cs

```diff
@@ -196,15 +196,25 @@ public static WebApiProject CreateNew(TestConfig configuration, WebApiProjectOpt
             return project;
         }
 
+        /// <summary>
+        /// Verifies if the specified file is part of the project.
+        /// </summary>
+        /// <param name="filename">The complete path and filename of the file, relative to the root folder of the project (without leading (back)slash).</param>
+        /// <returns>True if the file is present, otherwise false.</returns>
+        public bool ContainsFile(string filename)
+        {
+            return File.Exists(Path.Combine(ProjectDirectory.FullName, filename));
+        }
```

Comment:
This is probably nitpicking. I would add a guard check here to verify if it's not a blank file name 😄 .

Reply:
I'll add one.